why i am getting error in when i am pushing the data

this time i can see the data

